packages/common-library/src/common_library/pydantic_validators.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+from pydantic import field_validator
+
+
+def _try_convert_str_to_float_or_return(
```

Comment:
suggestion: `validate_numeric_string_as_timedelta`

Reply:
also why is this method private? how do you then use it? Is this convoluted thing below really necessary?